### PR TITLE
Fix Bug 1237070, implement lean data practices page

### PR DESCRIPTION
**Pull request by schalkneethling**
Waiting on one link, but the rest is open for code review. @craigcook r? Also note, this page is not localized and support is IE9 and above. As long as the content is readable in IE8 and lower, we are all good.

### CODE REVIEW

**Comment by schalkneethling** [comment]
Just noticed some of the SVG files can do with some optimization. Will update the PR with those.

**Comment by schalkneethling** [comment]
Ok, added the Github link and optimized the SVG files.

File: media/css/mozorg/about-lean-data.less [hunk 1]
```diff
@@ -0,0 +1,303 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+.circular-image(@color: #333, @width: 180px, @height: 180px) {
```
Comment (craigcook) [inline media/css/mozorg/about-lean-data.less]: Doesn't look like this mixin us used anywhere.

Comment (schalkneethling) [inline media/css/mozorg/about-lean-data.less]: Ah yes, we no longer have circular images, good catch. removing.

**Comment by craigcook** [comment]
r+ :rocket:

**Comment by schalkneethling** [comment]
Ok @craigcook, all the ducks are in a row, merge at will. Thx

**Pull request merged**